HACKING.md

```diff
@@ -39,3 +39,42 @@ To run the unit tests:
 To run the integration tests:
 
     ./integration-test.sh
+
+## Generating Stats
+
+To generate the stats on your local machine you will need to import the listens into HDFS. 
+You should use the ``listenbrainz-incremental-dumps`` for your local environment as the size of 
+``listenbrainz-full-export-dumps`` is large for a local machine (The latest 
+``listenbrainz-full-export-dump`` size is around 9.3 GB). Follow the given steps to import the 
+listens.
+
+### Update config.py
+
+Open the ``config.py`` file present in the ``listenbrainz_spark`` directory and update the FTP 
+listens directory to import the incremental dumps as given.
+
+    FTP_LISTENS_DIR = '/pub/musicbrainz/listenbrainz/incremental/'
+
+Now update the temporary listens directory and dump with the directory name and the .tar.xz file 
+you would like to import as temporary listens. e.g. To import the 
+``listenbrainz-listens-dump-127-20200210-000002-incremental`` refer the below-given code.
+
+    TEMP_LISTENS_DIR = 'listenbrainz-dump-127-20200210-000002-incremental/'
+    TEMP_LISTENS_DUMP = 'listenbrainz-listens-dump-127-20200210-000002-incremental.tar.xz'
+
+### Upload listens to HDFS
+
+Next open the ``bash`` terminal for  ``listenbrainz_playground_1`` container by executing the 
+following command
+
+    ./develop.sh spark exec playground bash
+
+Make sure you have the Listenbrainz Spark server running before opening the ``bash`` terminal for  
```

Comment:
`ListenBrainz Spark` would do. We don't really use server with it. Also, B capital in ListenBrainz everywhere :)

Reply:
Okay